### DISCUSSION TITLE
Improve export UI  ideas

### DISCUSSION THREAD

**Discussion by albertosaez** [dcm4che/dcm4chee-arc-light#2917] [Q&A] [answered]
Hi, 

Some ideas for improving the export function in the U:

> For large number of nodes, enable a "search" for the destinations.

![image](https://user-images.githubusercontent.com/3266240/102012633-cf9dd100-3d4b-11eb-9542-21b49ea1b808.png)

> Keep the last node used as destination the next time the export is selected, as in V2 (it saves a lot of time).

> Instead the circle, use some kind of modal window for showing the progress (and result). Similar to V2 solution,

**Comment by shral** [comment, marked as answer]
I opened an issue, for the two first points: https://github.com/dcm4che/dcm4chee-arc-light/issues/2923

The third point is problematic because the client (UI) doesn't have the information about the progress to be able to show a progress bar, it can (It should show already) the message when the process is finished (On the synchronized version). If the Study is very big, it will take very long to show the message, that's why for big studies one should use a qued exporter.
